src/background/index.js

```diff
@@ -1,56 +1,63 @@
-import { LOOKUP_PLACE, SCAN_PARAGRAPH } from '../extensionMessageTypes';
+import {
```

Comment:
style thing here.. but if its more than a few imports, the prefered way would be 
```
import * as constants from './constants.js';
```
and then everywhere below
```
constants. CONTEXT_MENU_TITLE
```

Reply:
Hmm, I was thinking of that, but what if we have more things in the file that we need? But for now that seems better. So I'll change it.